app/env.js

```diff
@@ -33,6 +33,8 @@ const vars = envalid.cleanEnv(
     METADATA_KEY_LENGTH: envalid.num({ default: 32 }),
     METADATA_VALUE_LITERAL_LENGTH: envalid.num({ default: 32 }),
     PROCESS_IDENTIFIER_LENGTH: envalid.num({ default: 32 }),
+    HEALTHCHECK_POLL_PERIOD_MS: envalid.num({ default: 30 * 1000, devDefault: 1000 }),
```

Comment:
the two new envs should be added to helm config too? Perhaps in a new PR

Reply:
and the README

Reply:
Mmm... why would helm need that? I assume that these are the default values and will set env vars anyway.

Timeout 5s and period to 30s

Reply:
We need to be able to set envars in K8S, hence including the envars in the helm templates.